rest-service/manager_rest/resources_v2.py

```diff
@@ -960,9 +962,18 @@ def list_events(self, query, include=None):
     def _search(self, query, include=None):
         es = ManagerElasticsearch.get_connection()
 
-        return es.search(index=self._set_index_name(),
-                         body=query,
-                         _source=include or True)
+        try:
+            return es.search(index=self._set_index_name(),
+                             body=query,
+                             _source=include or True)
+        except elasticsearch.TransportError as e:
+            # This happens when the events index is in the process
+            # of being created.
+            if (e.status_code == 503 and
+                    'SearchPhaseExecutionException' in e.error):
+                return {'hits': {'hits': [], 'total': 0}}
```

Comment:
I would go further and say that the implementation of `list_events` is very leaky as well. Anyway, I wish to avoid the refactoring such change will involve, at the moment.